content/en/blog/_posts/2022-XX-XX-iptables-chains.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: blog
+title: "Kubernetes’s IPTables Chains Are Not API"
+date: 2022-XX-XX
+slug: iptables-chains
+---
+
+### or, “Sorry, You Are Not Invited to SIG Network’s `MASQUERADE` Party”
+
+**Author:** Dan Winship (Red Hat)
+
+Some Kubernetes components (such as kubelet and kube-proxy) create
+IPTables chains and rules as part of their operation. These chains
```

Comment:
```suggestion
`iptable` chains and rules as part of their operation. These chains
```

Reply:
"iptables" is always plural.

It seems to me that we generally only write it as "`iptables`" when we are talking specifically about either the `/sbin/iptables` binary or the `mode: iptables` kube-proxy configuration option.

We probably say "iptables" more than "IPTables" in the existing docs, and I guess [upstream](https://www.netfilter.org/projects/iptables/index.html) refers to it without capital letters too...

Reply:
Oops, my typo. I meant to recommend lowercasing it only.

Reply:
Backticks are optional.